index.html

```diff
@@ -2127,6 +2134,9 @@ <h2>Capability Delegation</h2>
   <span class="comment">...</span>
 }
         </pre>
+
+        In the example above, the DID document for <code>did:example:123456789abcdefghi</code> defines two verificationMethods, <code>#keys-1</code> and <code>#keys-2</code>, that can be used for the purpose of capability delegation. This means that these verification methods can be used to produce and verify delegations of a capability issued to <code>did:example:123456789abcdefghi</code>.
```

Comment:
Hmm, this seems like it could be more clearly stated like this:

```suggestion
        In the example above, the DID document for <code>did:example:123456789abcdefghi</code> defines two verificationMethods, <code>#keys-1</code> and <code>#keys-2</code>, that can be used for the purpose of capability delegation. This means that these verification methods can be used to verify capabilities delegated by <code>did:example:123456789abcdefghi</code>.
```

Notably, a root capability may not be "issued" at all by an entity -- except perhaps conceptually from the target / system of authority, and yet if the controller were `did:example:123456789abcdefghi`, it could still be delegated by that entity using this verification method. Further, the verification method isn't really used to "produce" a delegated capability (aka "a delegation"), but rather to verify one.